### PR TITLE
Fixes/als 790 tracking UI

### PR DESCRIPTION
**Pull request by zeeshanmakeen**
ALS-790: iOS - Enable tracking UI issues.
ALS-791: iOS - Tracking Empty state UI issues 
ALS-802: [iOS] Tracker and tracker history issues
ALS-764: [iOS] Explore, tracking, geofence, navigation should not be a modal rather a stand-alone menu

### CODE REVIEW

File: LocationServices/LocationServices/Extensions/UIViewController+Extension.swift
Comment (denis-ryzhkov-makeen) [inline LocationServices/LocationServices/Extensions/UIViewController+Extension.swift:52]: let's not use force unwrap

File: LocationServices/LocationServices/Views/Common/CommonDashboardView.swift
Comment (denis-ryzhkov-makeen) [inline LocationServices/LocationServices/Views/Common/CommonDashboardView.swift:111]: debug code?

**Review by denis-ryzhkov-makeen — CHANGES_REQUESTED** [review]
The general improvements that should be applied:
- when drag the view down it is shown over tabbar. Should slide under.
- when we show keyboard expand the view to full height. When we shrink view from full large to medium the keyboard should disappear. (This are embellishments that are working out of the box for native sheet controller. Please, communicate this with Stas if we need to implement them)
- after closing add geofence screen, the geofence list is presented even if I've not opened it
- can't close poi card
- check with Stas if scroll of scrollviews and tableviews inside viewcontroller should expand/interract bottom sheet

File: LocationServices/LocationServices/Extensions/UIViewController+Extension.swift
Comment (zeeshanmakeen) [reply LocationServices/LocationServices/Extensions/UIViewController+Extension.swift:33]: this function is unused. removed

File: LocationServices/LocationServices/Coordinators/ChildCoordinators/TrackingCoordinator.swift
Comment (zeeshanmakeen) [reply LocationServices/LocationServices/Coordinators/ChildCoordinators/TrackingCoordinator.swift:76]: done

File: LocationServices/LocationServices/Coordinators/ChildCoordinators/TrackingCoordinator.swift
Comment (zeeshanmakeen) [reply LocationServices/LocationServices/Coordinators/ChildCoordinators/TrackingCoordinator.swift:145]: done

File: LocationServices/LocationServices/Extensions/UIViewController+Extension.swift
Comment (zeeshanmakeen) [reply LocationServices/LocationServices/Extensions/UIViewController+Extension.swift:83]: I think the short name is sufficient. Also, it is not the scope of this function to remove gesture. We can have another function but that is not needed anywhere right now.

File: LocationServices/LocationServices/Extensions/UIViewController+Extension.swift
Comment (zeeshanmakeen) [reply LocationServices/LocationServices/Extensions/UIViewController+Extension.swift:95]: changed & refactored

File: LocationServices/LocationServices/Extensions/UIViewController+Extension.swift
Comment (zeeshanmakeen) [reply LocationServices/LocationServices/Extensions/UIViewController+Extension.swift:52]: done

File: LocationServices/LocationServices/Views/Common/CommonDashboardView.swift
Comment (zeeshanmakeen) [reply LocationServices/LocationServices/Views/Common/CommonDashboardView.swift:111]: No it is asked not to show maybe later button. I've not removed it but just hid it

File: LocationServices/LocationServices/Extensions/UIViewController+Extension.swift
Comment (zeeshanmakeen) [reply LocationServices/LocationServices/Extensions/UIViewController+Extension.swift:33]: this function is unused. removed

File: LocationServices/LocationServices/Coordinators/ChildCoordinators/TrackingCoordinator.swift
Comment (zeeshanmakeen) [reply LocationServices/LocationServices/Coordinators/ChildCoordinators/TrackingCoordinator.swift]: done

> The general improvements that should be applied:
> 
> * when drag the view down it is shown over tabbar. Should slide under.
> * when we show keyboard expand the view to full height. When we shrink view from full large to medium the keyboard should disappear. (This are embellishments that are working out of the box for native sheet controller. Please, communicate this with Stas if we need to implement them)
> * after closing add geofence screen, the geofence list is presented even if I've not opened it
> * can't close poi card
> * check with Stas if scroll of scrollviews and tableviews inside viewcontroller should expand/interract bottom sheet

All Done

Comment (zeeshanmakeen) [reply LocationServices/LocationServices/Coordinators/ChildCoordinators/TrackingCoordinator.swift]: Will ask for review from stas once it is approved and build is provided

File: LocationServices/LocationServices/Views/Common/CommonDashboardView.swift
Comment (zeeshanmakeen) [reply LocationServices/LocationServices/Views/Common/CommonDashboardView.swift:111]: No it is asked not to show maybe later button. I've not removed it but just hid it

**Pull request merged**
